src/kayenta/report/list/table.tsx

```diff
@@ -12,7 +15,70 @@ import { PipelineLink } from './pipelineLink';
 
 import './executionList.less';
 
+const isAtlasScope = (scope: string, metrics: ICanaryMetricConfig[]) => (
```

Comment:
Could you please leave a comment here for future developers that suggests that any additional special handling for a metric store (e.g., for Stackdriver, Prometheus, or Datadog) could / should be factored out of this file?

Reply:
But then what incentive will I have to get this function out of here? :trollface:

I will indeed add a comment. Longer term I'd love it if we could figure out a better way to shape this location data on the atlas metric store side so it fit the model nicely instead of requiring this hack.